core/src/main/scala/dagr/core/execsystem/TaskManager.scala

```diff
@@ -553,23 +553,26 @@ class TaskManager(taskManagerResources: SystemResources = TaskManagerDefaults.de
       allDone = graphNodesInStatesFor(List(ORPHAN, COMPLETED)).size == graphNodes.size
 
       if (!allDone && runningTasks.isEmpty && tasksToSchedule.isEmpty) {
-        logger.error(s"There are ${readyTasks.size} tasks ready to be scheduled but not enough system resources available.")
-        readyTasks.foreach { readyTask =>
-          val resourcesType: String = readyTask match {
-            case _: FixedResources    => "FixedResources"
-            case _: VariableResources => "VariableResources"
-            case _: Schedulable       => "Schedulable"
-            case _                    => "Unknown Type"
+        if (readyTasks.nonEmpty) {
```

Comment:
@nh13 I'm more than happy to incorporate any messaging you want, but I'm not the author of this code (you are), and I'm honestly not sure what other cases this needs to handle.  

E.g. is it even actually possible to have tasks with unmet dependencies where the _upstream_ task is neither complete, nor "not schedulable in the available resources"?  If so, what state are they in?